packages/components/ContextualMenu/src/SubmenuItem.tsx

```diff
@@ -98,12 +99,18 @@ export const SubmenuItem = compose<SubmenuItemType>({
       },
       content: { children: text, testID },
       icon: createIconProps(icon),
+      chevron:{}
```

Comment:
nit: there should be a space between the colon and open bracket here. If you run prettier on the file, that should automatically get fixed for you.

If you have VSCode you can configure it to run formatting on save. There's instructions in the [main README](https://github.com/microsoft/fluentui-react-native#prettier).